### ISSUE TITLE
Legend shapeStyle

### ISSUE THREAD

**Issue by mzabriskie** [hshoff/vx#312]
`<Legend/>` `shapeStyle` prop is being implemented incorrectly by `Rect`.

https://github.com/hshoff/vx/blob/master/packages/vx-legend/src/shapes/Rect.js#L10

The `style` prop needs to be spread:

```js
style={{
  width,
  height,
  background: fill,
  ...style
}}
```

**Comment by hshoff** [comment]
🤦‍♂️ will get the fix published as soon as possible.

**Comment by hshoff** [comment]
Published as `@vx/legend@0.0.167`

**Comment by mzabriskie** [comment]
Thank you for the quick fix!
